src/main/scala/no/ndla/oembedproxy/service/OEmbedServiceComponent.scala

```diff
@@ -28,7 +28,7 @@ trait OEmbedServiceComponent extends LazyLogging {
       p.find(_.supports(url)) match {
         case None => throw new ProviderNotSupportedException(s"Could not find an oembed-provider for the url '$url'")
         case Some(provider) =>
-          ndlaClient.fetch[OEmbed](
+          ndlaClient.fetchWithForwardedAuth[OEmbed](
```

Comment:
Tror egentlig ikke vi har lyst til å forwarde auth her heller. Denne går til en hvilken som helst provider. f.ex youtube.com. Den går vel også muligens mot nye h5p tjenesten. Krever den auth?

Reply:
Nye H5P-tjenesten skal lene seg på våre tokens i produksjonsøyemed i hvert fall. Har egentlig ikke diskutert om de trenger det for visning... Det treffer vel Jonas i det han gjør nå. Kan fjerne det inntil videre da...